proknow-sdk-test/ProKnowApiTest.cs

```diff
@@ -84,15 +84,5 @@ public async Task GetConnectionStatusAsync_BadBaseUrl()
             Assert.IsFalse(connectionStatus.IsValid);
             Assert.AreEqual("Invalid or missing credentials", connectionStatus.ErrorMessage);
         }
-
-        [TestMethod]
-        public async Task GetConnectionStatusAsync_BogusCredentials()
-        {
-            var credentialsFile = Path.Combine(TestSettings.TestDataRootDirectory, "./bogus_credentials.json");
-            var proKnowApi = new ProKnowApi(TestSettings.BaseUrl, credentialsFile);
-            var connectionStatus = await proKnowApi.GetConnectionStatusAsync();
-            Assert.IsFalse(connectionStatus.IsValid);
-            Assert.AreEqual("Invalid or missing credentials", connectionStatus.ErrorMessage);
-        }
```

Comment:
Was this removed because it was failing? Which assertion failed?

Reply:
Line 94, Here is why. this uses the api route "/status" which is hardcoded to return a status of 200, if you look at auth.js line 357, it is also hardcoded to bypass authroization, so this test will always return a successful.